util/marketplace.py

```diff
@@ -22,15 +22,14 @@ def __init__(self, app_config):
 
     def get_account_number(self, user):
         email = user.email
-        account_number = entitlements.get_web_customer_id(user.id)
-        if account_number is None:
+        account_numbers = entitlements.get_web_customer_ids(user.id)
+        if len(account_numbers) == 0:
```

Comment:
See comment above. This won't work if a null value is returned.

Reply:
Ok, probably better to expect to return a null value. Other parts of the code may be expecting a null value as well anyways